table_position:
  dual_table        : [2.5, 2.4, 0.0]
  movable_table_1200: [4.5, 1.9, 0.0]
  movable_table_1500: [5.5, 1.9, 0.0]
  movable_table_1800: [6.5, 1.9, 0.0]